IbanValidation/IbanValidation.csproj

```diff
@@ -9,8 +9,9 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>IbanValidation</RootNamespace>
     <AssemblyName>IbanValidation</AssemblyName>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.5.2</TargetFrameworkVersion>
```

Comment:
Is there a reason why you upgraded the target framework version? I'd like to keep it as low as possible...